modify-semantic-analysis.js

```diff
@@ -0,0 +1,59 @@
+var path = require('path');
```

Comment:
this is already part of the other pr right?

Reply:
Good catch 😬